src/commands/edit-input-schema.js

```diff
@@ -113,7 +124,7 @@ class EditInputSchemaCommand extends ApifyCommand {
                 const inputSchemaObj = req.body;
                 let inputSchemaStr = JSON.stringify(inputSchemaObj, null, jsonIndentation);
                 if (appendFinalNewline) inputSchemaStr += '\n';
-                fs.writeFileSync(path, inputSchemaStr, { encoding: 'utf-8', flag: 'w+' });
+                fs.writeFileSync(inputSchemaPath, inputSchemaStr, { encoding: 'utf-8', flag: 'w+' });
```

Comment:
This will fail if you try to save into `.actor/INPUT_SCHEMA.json` (the new default), but there is no `.actor` directory. You'll have to do something like this:

```suggestion
                const path = require('path');
                const inputSchemaDir = path.dirname(inputSchemaPath);
                fs.mkdirSync(inputSchemaDir, { recursive: true });
                fs.writeFileSync(inputSchemaPath, inputSchemaStr, { encoding: 'utf-8', flag: 'w+' });
```

Reply:
good catch!